sapi/cgi/cgi_main.c

```diff
@@ -1957,6 +1957,7 @@ consult the installation file that came with this distribution, or visit \n\
 			 */
 			tsrm_shutdown();
 #endif
+			free(bindpath);
```

Comment:
Should here be efree(bindpath) maybe like suggested in the coding standards md file? This is maybe for some BSD system or in general? Thank you.

Reply:
It is allocated via classic memory allocation.